controllers/teams.js

```diff
@@ -0,0 +1,201 @@
+const Team = require( '../models/schemas/team' );
+const User = require( '../models/schemas/user' );
+
+exports.getTeams = ( req, res, next ) => {
+    Team.find( {}, ( err, teams ) => {

```

Comment:
Expect filters here, ie a user shouldn't see teams he's not assigned to unless he's an admin.

Reply:
I'll put `adminRequired` middleware before it, and make new route for `userTeams`, Sounds good?

Reply:
Yes, that's better.